l10n-pocket/en/footer.ftl

```diff
@@ -35,3 +36,21 @@ pocket-footer-cookie-preferences = Cookie preferences
 pocket-footer-read-it-later-brand-name = { -brand-name-read-it-later-inc }
 pocket-footer-twitter-brand-name = { -brand-name-twitter }
 pocket-footer-facebook-brand-name = { -brand-name-facebook }
+
+### Platform footer is used in some pages:
+
+### Pocket homepage https://www-dev-pocket.allizom.org
+### Add page URL https://www-dev-pocket.allizom.org/Add
+### All platform pages https://www-dev-pocket.allizom.org/ios
+### https://www-dev-pocket.allizom.org/android
+### https://www-dev-pocket.allizom.org/chrome
+### https://www-dev-pocket.allizom.org/safari
+### https://www-dev-pocket.allizom.org/opera
+### https://www-dev-pocket.allizom.org/edge
+
+pocket-footer-explore = Explore
+pocket-footer-publishers = Publishers
+pocket-footer-developers = Developers
+pocket-footer-support = Support
+pocket-footer-jobs = Jobs
+pocket-footer-read-it-later = 2022 Read it Later, Inc
```

Comment:
Suggestion: should `Read it Later, Inc` be a brand name?

Reply:
The year should probably be a variable and automated in the template, not hardcoded in the string ([bedrock footer, for reference](https://github.com/mozilla/bedrock/blob/main/bedrock/base/templates/includes/protocol/footer/footer.html#L101))

Reply:
@alexgibson yeah I had to look it up while coding this. Did some quick research and `Read it Later, Inc` is registered under Pocket: https://www.bloomberg.com/profile/company/1260798D:US

Reply:
@craigcook good looking out -- thanks!